ops/daniel-kit/0_restore.ps1

```
@@ -0,0 +1,49 @@
+<#
+.SYNOPSIS
+    Step 0: Restore PACS database from archived backups.
+.DESCRIPTION
+    Delegates to the proven restore pipeline. Creates tf-mssql container
+    with pacs_oltp database (112,057 Benton County properties).
+#>
+param(
+    [string]$SaPassword = "TF_Pacs2026!"
```

PSScriptAnalyzer (1.24.0)

[warning] Parameter '$SaPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $SaPassword type to SecureString; Set $SaPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)